src/Screen/Layouts/Block.php

```diff
@@ -37,6 +37,7 @@ abstract class Block extends Layout
     public function __construct(array $layouts = [])
     {
         $this->layouts = $layouts;
+        $this->variables['vertical'] = false;
```

Comment:
Let's use the property:

```php
protected $layouts = [
    'vertical' => false
];
```

Reply:
I tried this but I got `Undefined variable`

Reply:
Nevermind, the variable name was `$variables`